docs/mvvm/source-gen/diagnostics/MVVMTK0026.md

```diff
@@ -0,0 +1,33 @@
+---
+title: MVVM Toolkit error MVVMTK0026
+author: Sergio0694
+description: MVVM Toolkit error MVVMTK0026
+keywords: community toolkit, dotnet, csharp, mvvm, net core, net standard, source generators
+dev_langs:
+  - csharp
+---
+
+# MVVM Toolkit error MVVMTK0026
+
+Cannot apply `[NotifyDataErrorInfo]` to fields that don't have any validation attributes to use during validation.
+
+The following sample generates MVVMTK0026:
+
+```csharp
+using System.ComponentModel.DataAnnotations;
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace MyApp;
+
+public partial class SampleViewModel : ObservableValidator
+{
+    // This property is requesting validation but has no validation attributes
```

Comment:
Did we have the opposite where we'd put a warning for attributes detected besides validation ones that we don't recognize/handle forwarding on?

Reply:
Nope, we can't do that because devs might just be intentionally using attributes on fields, so we can't assume doing so is a mistake. Adding warnings there would potentially just add unintentional noise and might be confusing. It's a bit of an unfortunate situation, I don't think we can do much on that front. Hopefully it's just temporary until C# 12.